CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1655,7 +1764,7 @@ local function AddNewRow(pageToAddTo, rowDisplayName, selectionType, rowValues,
 	end
 
 	local RowFrame = nil
-	RowFrame = Util.Create'ImageButton'
+	RowFrame, RowFrameInfo = MakeCustomButton'ImageButton'
```

Comment:
RowFrameInfo is in the global name scope since you didn't specify local for it. This might cause some unintended behaviors, so lets move it to be a local variable too.